job-descriptions/software-engineer-security.md

```diff
@@ -2,28 +2,31 @@
 
 ## Responsibilities
 
 - Identify what our security needs are and develop a roadmap to improve and ensure the security of our product.
-- Execute on that roadmap through a mix of directly contributing to our codebase (i.e., Go, TypeScript, Kubernetes, Docker, Google Cloud Platform), and helping other engineers on our team make the necessary changes. You should expect that at least 75% of your effort will be direct contributions to start.
+- Execute on that roadmap through a mix of directly contributing to our codebase (i.e., Go, TypeScript, Kubernetes, Docker, Google Cloud Platform), and helping other engineers on our team make the necessary changes.
 - Promptly respond to and address security issues reported to us by third parties.
+- Increasing our security posture by running traditional security tools such as vulnerability scanners and SAST and DAST tools.
 - Create a culture of security at Sourcegraph that empowers all of our engineers to write secure code.
 
 ## Qualifications
 
-- Broad knowledge of possible attack vectors for on-premise and cloud applications, like Sourcegraph, and an understanding of how to eliminate and/or mitigate those security risks. This knowledge generally comes from having many years of experience in the domain of security.
-- Ability to define, plan, and prioritize security work that needs to be done (and then go do that work).
+- Experience identifying and remediating the full life cycle of vulnerabilities, both through external packages and in code.
+- Broad knowledge of attack vectors and threat actors for the life cycle of on-premise and cloud applications, including triage, remediation, and continuous improvement.
 - Ability to understand, debug, and write Go and TypeScript to fix security issues.
-- Ability to secure cloud applications that use our tech stack: Kuberentes, Docker, Google Cloud Platform.
+- Ability to secure cloud applications that use our tech stack: Kubernetes, Docker, Google Cloud Platform.
+- Ability to define, plan, and prioritize security work that needs to be done (and then go do that work).
 - Ability to communicate clearly and empathetically, especially in writing and documentation.
 
 ## Nice-to-haves
 
-- Experience leading security teams (as a people manager or as a technical leader).
 - Published blog posts and/or tech talks about your work.
 - Experience working on high-performing teams, preferably tech startups.
+- Experience improving the security posture of a small team, and helping that team carry the baton themselves.
+- Experience working in an ops environment.
```

Comment:
what does this mean?

Reply:
I'm trying to indicate that the ideal security candidate can experience managing infrastructure.  Now that I've stepped away from it a better point could be:
Experience managing and securing infrastructure